simapp/simd/cmd/testnet_test.go

```diff
@@ -0,0 +1,50 @@
+package cmd
```

Comment:
I made the test in `package cmd` so it could access the un-exported `testnetCmd` function. Is it better to change the method to be exported and put the test under `package cmd_test`?

Reply:
I prefer this way than to export the method

Reply:
Looks fine for this test. `package cmd_test` is used to avoid import cycles. It is good to be mindful that simapp imports everything, so importing simapp can easily lead to import cycles. Don't see a reason why it would happen at the `cmd` level though